RELEASE_PROCESS_OPENSEARCH.md

```diff
@@ -0,0 +1,313 @@
+
+### Distribution Build
+
+Refer the [Build Workflows](#build-workflows) section to get the details about the distribution build jobs. This section covers how to sequentially
+handle the distribution build job during the release.
+
+Following is the order of execution for the distribution build:
+
+##### OpenSearch
+```
+OpenSearch 
+OpenSearch + common-utils + job-scheduler 
+OpenSearch + common-utils + job-scheduler + performance-analyzer 
+OpenSearch + common-utils + job-scheduler + performance-analyzer + security 
+OpenSearch + common-utils + job-scheduler + ml-commons + performance-analyzer + security 
+All components (which are ready after completion of version increment)
+```
+
+##### OpenSearch Dashboards
+```
+OpenSearch Dashboards
+All components (which are ready after completion of version increment)
+```
+
+### Release Candidate
+
+The automated build workflows will create release candidates with the finalized commit and publish the docker images and associated distribution artifacts for testing. Now once all the version increment PR’s are completed and all the components are part of the input manifest, now its time to generate the RC.
+
+
+#### Sample Build details:
+
+| OpenSearch | OpenSearch Dashboards |
+|----------|----------|
+| [build_7858](https://build.ci.opensearch.org/job/distribution-build-opensearch/7848/console) | [build_6126](https://build.ci.opensearch.org/job/distribution-build-opensearch-dashboards/6126/) |
+
+The distribution build number denotes the RC, now with the above example finalized RC’s are `OS: 7848`, `OSD: 6126`.
+
+#### Integ Test Tar:
+
+| Integ Test Tar | x64 | arm64 |
+|----------|----------|----------|
+| OpenSearch  | [x64](https://build.ci.opensearch.org/job/integ-test/4906/console)  | [arm64](https://build.ci.opensearch.org/job/integ-test/4897/console)   |
+| OpenSearch Dashboards  | [x64](https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/3531/console)  | [arm64](https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/3530/console)   |
+
+#### Integ Test RPM:
+
+| Integ Test RPM | x64 | arm64 |
+|----------|----------|----------|
+| OpenSearch  | [x64](https://build.ci.opensearch.org/job/integ-test/4908/console)  | [arm64](https://build.ci.opensearch.org/job/integ-test/4907/console)   |
+| OpenSearch Dashboards  | [x64](https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/3532/console)  | [arm64](https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/3533/console)   |
```

Comment:
We can ignore the rpm ones for now as the tar is enough at this stage.
Rpm is similar to tar and deb is having a small bug at the moment.

Reply:
Sure Peter, we should still mention, we might get some contributions/fixes from external RM's. WDYT?